package.json

```diff
@@ -106,6 +114,100 @@
+    "babel-plugin-transform-es2015-classes": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-computed-properties": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-destructuring": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-duplicate-keys": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-for-of": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-function-name": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-literals": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-modules-amd": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-modules-commonjs": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-modules-systemjs": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-modules-umd": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-object-super": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-parameters": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-shorthand-properties": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-spread": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-sticky-regex": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-template-literals": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-typeof-symbol": "^7.0.0-beta.0",
+    "babel-plugin-transform-es2015-unicode-regex": "^7.0.0-beta.0",
+    "babel-plugin-transform-es3-member-expression-literals": "^7.0.0-beta.0",
+    "babel-plugin-transform-es3-property-literals": "^7.0.0-beta.0",
+    "babel-plugin-transform-exponentiation-operator": "^7.0.0-beta.0",
+    "babel-plugin-transform-export-extensions": "^7.0.0-alpha.0",
+    "babel-plugin-transform-flow-strip-types": "^7.0.0-beta.0",
+    "babel-plugin-transform-function-bind": "^7.0.0-beta.0",
+    "babel-plugin-transform-object-assign": "^7.0.0-beta.0",
+    "babel-plugin-transform-object-rest-spread": "^7.0.0-beta.0",
+    "babel-plugin-transform-react-constant-elements": "^7.0.0-beta.0",
+    "babel-plugin-transform-react-display-name": "^7.0.0-beta.0",
+    "babel-plugin-transform-react-jsx-self": "^7.0.0-beta.0",
+    "babel-plugin-transform-react-jsx-source": "^7.0.0-beta.0",
+    "babel-plugin-transform-react-jsx": "^7.0.0-beta.0",
+    "babel-plugin-transform-regenerator": "^7.0.0-beta.0",
+    "babel-plugin-transform-runtime": "^7.0.0-beta.0",
+    "babel-plugin-transform-strict-mode": "^7.0.0-beta.0",
+    "babel-polyfill": "^7.0.0-beta.0",
+    "babel-preset-es2015": "^7.0.0-beta.0",
+    "babel-preset-es2016": "^7.0.0-beta.0",
+    "babel-preset-flow": "^7.0.0-beta.0",
+    "babel-preset-react": "^7.0.0-beta.0",
+    "babel-preset-stage-0": "^7.0.0-beta.0",
+    "babel-preset-stage-1": "^7.0.0-beta.0",
+    "babel-preset-stage-2": "^7.0.0-beta.0",
+    "babel-preset-stage-3": "^7.0.0-beta.0",
+    "babel-register": "^7.0.0-beta.0",
+    "babel-runtime": "^7.0.0-beta.0",
+    "babel-template": "^7.0.0-beta.0",
+    "babel-traverse": "^7.0.0-beta.0",
+    "babel-types": "^7.0.0-beta.0"
+  },
```

Comment:
Force Yarn to ignore all version requirements for these packages and use the 7 beta instead. There’s probably too many overrides here, need to trim down.